tests/foreman/cli/test_hammer.py

```diff
@@ -196,3 +199,22 @@ def test_positive_disable_hammer_defaults(self):
             Defaults.delete({'param-name': 'organization_id'})
             result = ssh.command('hammer defaults list')
             self.assertTrue(default_org['id'] not in "".join(result.stdout))
+
+    @tier1
+    @run_in_one_thread
+    def test_hammer_host_info_output(self):
+        """Verify re-add of 'owner-id' in `hammer host info` output
+
+        :id: 03468516-0ebb-11eb-8ad8-0c7a158cbff4
+
+        :Steps:
+            1. Update the host with any owner
+            2. Get host info by running `hammer host info`
+
+        :expectedresults: 'owner-id' should be in `hammer host info` output
+
+        :BZ: 1779093
+        """
+        Host.update({'owner': settings.server.admin_username, 'owner-type': 'User', 'id': '1'})
+        result = Host.info({'id': '1', 'fields': 'Additional info'})
+        assert 'Owner Id' in str(result)
```

Comment:
Would it be good idea to also verify `Owner Id` field value, not only label presence?

Reply:
I too think value should be verified but then again it goes out of the scope of the bug. Should I?

Reply:
Up to you, but if you think it's good idea, then yes, you should. Nobody says our test must cover **only** bug - it can cover bug fix and some things around, especially when they are cheap and easy to check.

Reply:
Implemented verification of `owner-id`.